src/mozilla_sec_eia/models/sec10k/entities.py

```diff
@@ -0,0 +1,58 @@
+"""Define table structure SEC10k extraction."""
```

Comment:
I've created [pandera](https://docs.dagster.io/integrations/pandera) models for all of our output tables, which provides some documentation on schema, and dagster checks/enforces table structure.